src/app/results/results.component.css

```diff
@@ -365,41 +365,141 @@ div.autocorrect{
   margin-left: 6%;
 }
 
-@media screen and (min-width:1920px) {
-  #tools {
-    margin-left: 27%;
+
+@media screen and (max-width: 1920px) {
+  #search-options {
+    margin-left: 5.5%
+  }
+  div.result{
+    margin-left: -5.7%;
+  }
+  div.autocorrect{
+    margin-left: 0;
+  }
+  div.result.message-bar{
+    margin-left: 0;
+  }
+  div.feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
+    width: 34%;
   }
 }
 
-@media screen and (max-width: 1440px) {
-  .feed {
-    width: 43%;
+@media screen and (max-width: 1800px) {
+  #search-options {
+    margin-left: 6.5%
+  }
+  div.result{
+    margin-left: -2.7%;
+  }
+  div.autocorrect{
+    margin-left: 1%;
+  }
+  div.result.message-bar{
+    margin-left: 1%;
+  }
+  div.feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
+    width: 36%;
   }
 }
+@media screen and (max-width: 1750px){
```

Comment:
@nikhilrayaprolu I tried this, it worked to some extent, but see this
Everything is perfectly aligned at 1023px
![image](https://user-images.githubusercontent.com/20185076/28585167-03884cea-718d-11e7-8644-203d572a872a.png)
It all gets messed up at 778px
![image](https://user-images.githubusercontent.com/20185076/28585195-1f626e5a-718d-11e7-89b8-b557a13434fb.png)
And there is no break point in between.